src/ShallowTraversal.js

```diff
@@ -66,6 +69,18 @@ export function nodeHasId(node, id) {
   return propsOfNode(node).id === id;
 }
 
+
+export function nodeHasProperty(node, propKey, propValue) {
+  const nodeProps = propsOfNode(node);
+
+  if (propValue) {
+    return nodeProps[propKey] === propValue;
```

Comment:
`===`. A prop of `3` and a prop of `'3'` are different and shouldn't be selected in the same group.